src/views/Resources.vue

```diff
@@ -3,51 +3,98 @@
     <v-row justify="center">
       <v-col cols="10" max-width="1600">
         <v-card class="elevation-12">
-          <div class="text-center">
-            <h1>Resources table</h1>
-          </div>
-          <div class="d-flex justify-end">
-            <v-col cols="12" sm="6" md="4" class="text-end">
-              <v-btn rounded="xs" size="x-large" color="red" to="/resources/add"
-                >Add resource</v-btn
-              >
-            </v-col>
+          <v-container class="text-center text-h4 font-weight-bold">
+            {{
+              sortChoice === "All"
+                ? "All resources table"
+                : `${sortChoice}'s resources table`
+            }}
+          </v-container>
+          <div class="d-flex justify-space-between">
+            <v-menu>
+              <template v-slot:activator="{ props }">
+                <v-btn
+                  variant="outlined"
+                  class="mx-4"
+                  rounded="xs"
+                  :size="isSmAndDown ? 'small' : 'x-large'"
+                  color="red"
+                  v-bind="props"
+                >
+                  Sorting
+                </v-btn>
+              </template>
+              <v-list>
+                <v-list-item
+                  v-for="(item, index) in items"
+                  :key="index"
+                  :value="index"
+                >
+                  <v-list-item-title @click="clickHandler(item.title)">{{
+                    item.title
+                  }}</v-list-item-title>
+                </v-list-item>
+              </v-list>
+            </v-menu>
+            <v-btn
+              class="mx-4"
+              rounded="xs"
+              :size="isSmAndDown ? 'small' : 'x-large'"
+              color="red"
+              to="/resources/add"
+              >Add resource</v-btn
+            >
           </div>
 
-          <resource-table></resource-table>
+          <resource-table :sortChoice="sortChoice"></resource-table>
         </v-card>
       </v-col>
     </v-row>
   </v-container>
 </template>
 
 <script>
-import { onMounted, inject } from "vue";
+import { onMounted, inject, ref, computed } from "vue";
 import ResourceTable from "@/components/Table/ResourceTable.vue";
 import { useStore } from "vuex";
+import { useDisplay } from "vuetify/lib/framework.mjs";
 
 export default {
   components: {
     ResourceTable,
   },
   setup() {
     const store = useStore();
-    const showSnackbar = inject("showSnackbar");
+    const snackbarProvider = inject("snackbarProvider");
+    const mobile = useDisplay();
```

Comment:
this name is misleading. I would suggest removing this variable from here altogether and when using it in the isSmAndDown call it directly:

```
 isSmAndDown: computed(() => {
         return useDisplay().smAndDown.value;
       }),
```